tests/Issues/Issue779Test.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Issues;
+
+use Mpdf\Mpdf;
+use Mpdf\Output\Destination;
+
+class Issue779Test extends \Mpdf\BaseMpdfTest
+{
+	public function testOffsetsInHtmlTable()
+	{
+		$content = file_get_contents(__DIR__ . '/../data/html/issue779.html');
+		$configs = [
```

Comment:
Maybe lose these altogether? They actually have nothing to do with the error so they can be left out.